### PR TITLE
added touchpoints form code

### PR DESCRIPTION
**Pull request by nick-mon1**
- added new touchpoints code and tested URLs
- refactored button styles and template setup

**Before**

Screen Shot 2022-07-08 at 11 24 13 AM

**After**

Screen Shot 2022-07-08 at 11 23 48 AM

### CODE REVIEW

**Comment by nick-mon1** [comment]
In order to keep the visibility logic on sub-pages, the button is located in the `layouts/partials/core/page-data.html`.
Using `position: sticky` will limit the button to stay within the main content column that spans 9 columns.

This moves the button over to the left, could be position to the right of the parent container but will require additional coding if we want this be sticky and moved next to the edit button.

File: themes/digital.gov/layouts/partials/core/touchpoints-form.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<script src="https://touchpoints.app.cloud.gov/touchpoints/dc4265f6.js" async></script>
+<!-- <section class="touchpoints-form-container" style="position:fixed;bottom:35px;right:100px;z-index:500;">
+  <button id="touchpoints-modal-button" class="usa-button usa-button--outline" type="button">Was this page helpful?</button>
+</section> -->
```
Comment (mejiaj) [inline themes/digital.gov/layouts/partials/core/touchpoints-form.html:4]: Do we need this commented code?

Comment (nick-mon1) [reply themes/digital.gov/layouts/partials/core/touchpoints-form.html]: Commented code is removed ;)

**Review by mejiaj — APPROVED** [review]

**Review by mejiaj — DISMISSED** [review]

**Comment by ToniBonittoGSA** [comment]
Temp hold; Trello

**Comment by nick-mon1** [comment]
New requested style changes for the form button.

**Before**
touchpoints-desktop
touchpoints-mobile


**After**

Screen Shot 2022-07-14 at 10 12 51 AM
Screen Shot 2022-07-14 at 10 18 10 AM

**Review by ToniBonittoGSA — APPROVED** [review]

**Pull request merged**